Program.cs

```diff
@@ -94,10 +84,10 @@ static int PlaceObstacles(char[,] grid)
             column = random.Next(gridColumns);
         } while (grid[row, column] == '$' || grid[row, column] == 'P');
 
-        grid[row, column] = '|';
+        grid[row, column] = itemSymbol;
     }
 
-    return numberOfObstacles;
+    return numberOfItems;
 }
 
 static (int, int) PlacePlayer(char[,] grid)
```

Comment:
When you're not using custom seed, you should use `Random.Shared` instance instead of new().

Reply:
I think you're trying to break all the rules in the guidelines 🤔 
Never define variables side to side: `int playerRow, playerColumn;` should be 
```
int playerRow;
int playerColumn;
```

Reply:
Ad. MovePlayer() - formatting issue, either all arguments should be on the same line or each one should be on a separate line. Again, see the guidelines.

Reply:
Also in that method. L154 - what if you run into a monster? You're rewriting its position.

Reply:
Ad. MoveMonster(), again all `if` statements should user brackets, e.g. 
```
if ()
{
}
else 
{
}
```